lambda/aws-qa-appsync-opensearch/question_answering/src/llms/text_generation_llm_selector.py

```diff
@@ -50,9 +50,9 @@ def get_llm(callbacks=None):
 
     return Bedrock(**kwargs)
 
-def get_embeddings_llm():
+def get_embeddings_llm(model_id):
     bedrock = boto3.client('bedrock-runtime')
-    return BedrockEmbeddings(client=bedrock, model_id="amazon.titan-embed-text-v1")
+    return BedrockEmbeddings(client=bedrock, model_id=model_id)
     
 def get_max_tokens():
```

Comment:
this method needs to be updated as well, max number of tokens needs to be retrieved per model id, here it is hardcoded as we were using Claude

Reply:
done - 6cbb5e0c8ba77186876a170ee7fc58a5f4335ccb